style.css

```diff
@@ -0,0 +1,12 @@
+div {

```

Comment:
This is too general.  Determine what class of div's it should apply to.  Div's get used too generically to have a rule this general.

Reply:
It was for centering the page with wraparound text.  I'll remove it for now since the board is the only thing being used atm.